schemas/network.yml

```diff
@@ -16,11 +16,20 @@
           type: keyword
           description: >
             Name given by operators to sections of their network.
-
-    - name: protocol
+    - name: network
+      type: keyword
+      description: >
+          OSI Layer 3 Network Layer. Examples - IP, ICMP
```

Comment:
Could you add a bit more details to all descriptions. For example add/describe a use case from which you pick the fields.